lib/friendly_shipping/services/ship_engine.rb

```diff
@@ -36,7 +36,7 @@ def carriers
       def rate_estimates(shipment, carriers)
         request = FriendlyShipping::Request.new(
           url: API_BASE + 'rates/estimate',
-          body: SerializeRateEstimate.new(shipment: shipment, carriers: carriers).call.to_json,
+          body: SerializeRateEstimate.call(shipment: shipment, carriers: carriers).to_json,
```

Comment:
Can't this be `SerializeRateEstimate.(shipment: shipment, carriers: carriers).to_json`?

Reply:
This could be, but Rubocop complains about and I tend to agree, because the `.()` shortcut for `.call()` is not widely known and somehow weird syntax, no?

Reply:
Agreed. I just prefer consistency. The `.()` shortcut is already being used in `lib/friendly_shipping/services/ship_engine.rb` line 41.